File: Unidad_III/Brandon/ejemplo_2/Python/Auto/src/agencia.py
import sys
sys.path.append("../include")
from auto import Auto

auto1 = Auto()

pMarca=input("Ingrese la marca:  ")
pModelo=input("Ingrese el modelo:  ")
pYear=input("Ingrese el año:  ")
pKilo=input("Ingrese el Kilometraje:  ")
pKilo=int(pKilo)
pColor=input("Ingrese el color:  ")

auto2 = Auto(pMarca,pModelo,pYear,pKilo,pColor)

auto1.Checar()
auto2.Checar()

pColor=input("Ingrese el nuevo color: ")
auto2.Pintar(pColor)

pKilo=input("Cuanto corrio el auto? ")
pKilo=int(pKilo)
auto2.Correr(pKilo)

pModelo=input("Cual es el nuevo modelo: ")
auto2.Modificar(pModelo)

auto2.Checar()

File: Unidad_III/Brandon/ejemplo_2/C++/Auto/src/Agencia.cpp
#include <auto.hpp>

int main()
{
    std::string cMarca,cModelo,cColor;
    int cYear, cKilo;

    Auto auto1;

    std::cout << "Ingrese la marca" << std::endl;
    std::cin >> cMarca;
    std::cout << "Ingrese el modelo" << std::endl;
    std::cin >> cModelo;
    std::cout << "Ingrese el año" << std::endl;
    std::cin >> cYear;
    std::cout << "Ingrese el kilometraje" << std::endl;
    std::cin >> cKilo;
    std::cout << "Ingrese el color" << std::endl;
    std::cin >> cColor;

    Auto* auto2 = new Auto(cMarca,cModelo,cYear,cKilo,cColor);

    auto1.Checar();
    auto2->Checar();

    std::cout << "Ingrese el nuevo color" << std::endl;
    std::cin >> cColor;
    auto2->Pintar(cColor);

    std::cout << "Ingrese cuanto corrió" << std::endl;
    std::cin >> cKilo;
    auto2->Correr(cKilo);

    std::cout << "Ingrese el nuevo modelo" << std::endl;
    std::cin >> cModelo;
    auto2->Modificar(cModelo);

    auto2->Checar();


return 0;
}